app/recordtransfer/forms/admin_forms.py

```diff
@@ -1,16 +1,76 @@
 """Forms specific to the recordtransfer admin site."""
 
-from typing import ClassVar
+from typing import Any, ClassVar
 
 from django import forms
+from django.contrib.auth.forms import UserChangeForm
 from django.utils.html import format_html
 from django.utils.translation import gettext
 
+from recordtransfer.forms.mixins import ContactInfoFormMixin
 from recordtransfer.models import (
     Submission,
+    User,
 )
 
 
+class UserAdminForm(ContactInfoFormMixin, UserChangeForm):
+    """Custom form for User admin that includes contact information fields."""
+
+    class Meta:
+        """Meta class for UserAdminForm."""
+
+        model = User
+        fields = "__all__"
+
+    CONTACT_FIELDS: ClassVar[list[str]] = [
+        "phone_number",
+        "address_line_1",
+        "city",
+        "province_or_state",
+        "postal_or_zip_code",
+        "country",
+    ]
+
+    READONLY_FIELDS: ClassVar[list[str]] = ["date_joined", "last_login"]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Make contact info fields not required for admin
+        for field_name in self.CONTACT_FIELDS:
+            if field_name in self.fields:
+                self.fields[field_name].required = False
+
+        # Set readonly fields
+        for field_name in self.READONLY_FIELDS:
+            self.fields[field_name].disabled = True
+            self.fields[field_name].widget = forms.DateTimeInput(attrs={"readonly": True})
```

Comment:
If you try to Save the form in the admin, you get this error:

![Screenshot 2025-06-27 205244](https://github.com/user-attachments/assets/cd19ede1-2cb9-4954-a73d-5eeda6b30572)

The `DateTimeInput` widget is incompatible with those fields. I'd recommend just removing this widget and letting it use the default one. You're still able to click "Now" or change the time, but when you save the form, those fields aren't actually saved.

```suggestion
```